src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts

```diff
@@ -282,7 +286,6 @@ export class ReviewZoneWidget extends ZoneWidget implements ICommentThreadWidget
 			}
 		} else {
 			this._commentThread.collapsibleState = languages.CommentThreadCollapsibleState.Expanded;
-			this.show({ lineNumber: lineNumber, column: 1 }, 2);
 		}
```

Comment:
Is it required for this PR that the `this.show` line is removed? I know that https://github.com/microsoft/vscode/issues/174263 refers to this, but now the collapse is not symmetric with the expand.

Reply:
Until I removed it the beak was getting put to the start of the line because that call to `this.show` hardwired `column: 1`. I briefly considered making callers of `toggleExpand` pass the `Range` instead of just the line number. But since it turned out that the call is superfluous I decided to remove it.

The line 283 call to `this.hide` is probably superfluous for the same reason, but it wasn't causing me a problem. Would you like me to remove it for symmetry?

Reply:
Thanks for confirming that the line removal is needed for this change. I'll take a look at `this.hide` as a part of https://github.com/microsoft/vscode/issues/174263.